hack/generate.sh

```diff
@@ -110,22 +126,31 @@ run_target() {
         done
         
         if [[ ${#invalid_options[@]} -gt 0 ]]; then
-            printf "Skipping invalid options: %s, Available options are: %s\n\n" "${invalid_options[*]}" "${available_options[*]}"
+            printf "Skipping invalid options: %s, Available options are: %s\n\n" "${invalid_options[*]}" "${AVAILABLE_CODEGEN_OPTIONS[*]}"
         fi
       fi
 
       if [[ ${#valid_options[@]} -gt 0 ]]; then
-        printf "\n> Generating codegen for groups: ${valid_options[*]}\n\n"
+        printf "\n> Generating codegen for groups: %s\n" "${valid_options[*]}"
         $REPO_ROOT/hack/update-codegen.sh --which "${valid_options[*]}" --mode "$MODE"
+      else
+        printf "!! No valid groups provided for codegen, Available groups are: %s\n\n"  "${AVAILABLE_CODEGEN_OPTIONS[*]}"
```

Comment:
Same, `exit 1` instead of continuing?

Reply:
My idea was to continue to run other targets even if this one fails, but maybe let's exit to make the error more visible